integration-tests/tests/model_arimax_test.py

```diff
@@ -0,0 +1,134 @@
+# vim: set encoding=utf-8
+
+#
+#  Copyright (c) 2016 Intel Corporation 
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+import unittest
+import trustedanalytics as ta
+
+# show full stack traces
+ta.errors.show_details = True
+ta.loggers.set_api()
+
+if ta.server.port != 19099:
+    ta.server.port = 19099
+ta.connect()
+
+class ModelArimaxTest(unittest.TestCase):
+    def test_arimax_with_lag(self):
+        print "define csv file"
+        schema = [("y", ta.float64),("visitors", ta.float64),("wkends", ta.float64),("seasonality", ta.float64),("incidentRate", ta.float64), ("holidayFlag", ta.float64),("postHolidayFlag", ta.float64),("mintemp", ta.float64)]
```

Comment:
I have a pull request that's changing the arx_train/arx_test data to use air quality (since Kyle said not to use this in examples/tests), so don't use those csv files in your tests.  Thanks